vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "editor.formatOnSave": true,
+  "eslint.validate": ["typescript"],
+  "editor.codeActionsOnSave": {
+    "source.fixAll": true
+  }
+}
```

Comment:
This file also looks like the one for personal use :) However, you might come from a team with another code style agreements. I am curious to hear, if that is so :)

Reply:
This file is not some team's code style agreements. I will move it to gitignore